### DISCUSSION TITLE
Q values and MBR

### DISCUSSION THREAD

**Discussion by vmunro2** [vdemichev/DiaNN#677] [Q&A] [answered]
Hi,

I have some questions about q values, using the main report and MBR. 

1. Which q value in the main report is being filtered depending on the precursor FDR in the GUI? Does that apply to the report or the generation of a spectral library or both?
2. I want to generate my own protein report from the main report, but I am confused about which q values to filter on. If I generate a spectral library by performing a library free search in DIA-NN using a subset of my samples (with MBR off), and use this to search all of my data, should I use ‘Q.Value’ as the run-specific q value, and ‘Lib.Q.Value’ and ‘Lib.PG.Q.Value’ as the global q-values? i.e essentially a manual MBR?
3. When using MBR, why does the Global.Q.Value change between first and second pass? (even though we are supposed to use Lib.Q.Value)

Thank you!

**Comment by vdemichev** [comment, marked as answer]
1. Q.Value column in both main report and the spectral library being generated. In the case of a library it has a different meaning: not run-specific q-value but global precursor q-value.
2. Q.Value <= 0.01 & Lib.Q.Value <= 0.01 & Lib.PG.Q.Value <= 0.01 is the standard way of doing this for most experiments. PG.Q.Value <= 0.01 in addition to that ensures run-specific protein FDR, but this is often not used as it reduces data completeness.
3. Because second pass is performed using same algorithms that would have been used with any DIA library, not necessarily generated from the same data. In this case, global q-values are penalised depending on the library q-value to ensure correct FDR control, meaning that Global.Q.Value will be more strict than Lib.Q.Value. If the DIA library however was created from the same data, as is the case with MBR, only makes sense to use the Lib.Q.Value.

Best,
Vadim

**Reply by vmunro2** [reply]
Great, thank you very much!
